.github/workflows/pull.yml

```diff
@@ -172,6 +172,8 @@ jobs:
           { config: "distributed", shard: 2, num_shards: 3, runner: "linux.8xlarge.nvidia.gpu" },
           { config: "distributed", shard: 3, num_shards: 3, runner: "linux.8xlarge.nvidia.gpu" },
           { config: "functorch", shard: 1, num_shards: 1, runner: "linux.4xlarge.nvidia.gpu" },
+          { config: "deploy", shard: 1, num_shards: 1, runner: "linux.4xlarge.nvidia.gpu" },
+          { config: "deploy", shard: 1, num_shards: 1, runner: "linux.4xlarge.nvidia.gpu" },
```

Comment:
do we need GPUs for these tests? or is CPU sufficient?

Reply:
We only need 1 gpu (good catch on that). However, we are planning on adding gpu tests to do this right, for which we'd want the gpu. We talked about this offline.